### PR TITLE
Pull request for ClassicPress Migration Plugin

### PR DESCRIPTION
**Pull request by marctenax**
OK, this is all my optimized coding, and best practice, for this plugin. Thanks!

### CODE REVIEW

File: switch-to-classicpress-from-wordpress.php [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+ * Load MU-Plugin (dir) Textdomain.
+ */
+function classicpress_load_muplugin_textdomain() {
+	load_muplugin_textdomain( 'switch-to-classicpress-from-wordpress', basename( dirname( __FILE__ ) ) . '/languages' );
+}
+
+/**
+ * Adds Plugin Row Meta Links.
+ */
+function classicpress_adds_row_meta_links( $plugin_meta, $plugin_file ) {
```
Comment (scottybo) [inline switch-to-classicpress-from-wordpress.php]: Please remove this - I don't want to promote the petition from the plugin, I think it's safe to say Gutenberg is coming so we should focus on this plugin's core functionality.

File: switch-to-classicpress-from-wordpress.php [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+	header( 'Connection: Close'      );
+		exit;
+}
+
+/**
+ * Currently Plugin Version.
+ *
+ * Start at version 1.0.0 and use SemVer - https://semver.org/
+ */
+define( 'SWITCH_TO_CLASSICPRESS_FROM_WORDPRESS_VERSION', '1.0.2' );
```
Comment (scottybo) [inline switch-to-classicpress-from-wordpress.php]: Please keep at v 1.0.0

File: switch-to-classicpress-from-wordpress.php [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+	header( 'HTTP/1.0 403 Forbidden' );
+	header( 'HTTP/1.1 403 Forbidden' );
+	header( 'HTTP/2.0 403 Forbidden' );
+	header( 'Status:  403 Forbidden' );
+	header( 'Connection: Close'      );
+		exit;
+}
+
+/**
+ * Currently Plugin Version.
```
Comment (scottybo) [inline switch-to-classicpress-from-wordpress.php]: Typo, should be "Current plugin version"

File: switch-to-classicpress-from-wordpress.php [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+Requires WP:       4.9
+*/
+
+	/**
+	 * Switch to ClassicPress from WordPress.
+	 *
+	 * ClassicPress Migration Plugin.
+	 *
+	 * @package Switch_to_ClassicPress_from_WordPress
+	 * @version 1.0.2 - 2018-08-30
```
Comment (scottybo) [inline switch-to-classicpress-from-wordpress.php]: Please remove :)

File: switch-to-classicpress-from-wordpress.php [hunk 5]
```diff
@@ -0,0 +1,106 @@
+<?php 
+/*
+Plugin Name:       Switch to ClassicPress from WordPress
+Plugin URI:        https://github.com/classicpress/classicpress-migration-plugin/
+Description:       Switch to ClassicPress from WordPress and say Goodbye to Gutenberg.
+Version:           1.0.2
```
Comment (scottybo) [inline switch-to-classicpress-from-wordpress.php]: Should be 1.0.0

File: switch-to-classicpress-from-wordpress.php [hunk 6]
```diff
@@ -0,0 +1,106 @@
+<?php 
+/*
+Plugin Name:       Switch to ClassicPress from WordPress
+Plugin URI:        https://github.com/classicpress/classicpress-migration-plugin/
```
Comment (scottybo) [inline switch-to-classicpress-from-wordpress.php]: Incorrect URL

File: switch-to-classicpress-from-wordpress.php [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+Description:       Switch to ClassicPress from WordPress and say Goodbye to Gutenberg.
```
Comment (scottybo) [inline switch-to-classicpress-from-wordpress.php]: Please remove "and say Goodbye to Gutenberg"

File: readme.txt [hunk 8]
```diff
@@ -0,0 +1,36 @@
...
+
+Disable Gutenberg call to action inside the dashboard.
+
+== Changelog ==
+
+= 1.0.2 =
+
+* Change slug and name according to ClassicPress community.
+
+= 1.0.1 =
```
Comment (scottybo) [inline readme.txt]: Please remove versions after 1.0.0 - we're still on the commits for the initial release

File: readme.txt [hunk 9]
```diff
@@ -0,0 +1,36 @@
...
+* Extended code and functionality.
+
+= 1.0.0 =
+
+* Initial release.
+* Disable Gutenberg call to action inside the dashboard.
+
+== Upgrade Notice ==
+
+= 1.0.2 =
```
Comment (scottybo) [inline readme.txt]: Please remove

File: readme.txt [hunk 10]
```diff
@@ -0,0 +1,36 @@
+=== Switch to ClassicPress from WordPress ===
+Contributors: classicpress
+Tags: classicpress, classic-press, classic, press
```
Comment (scottybo) [inline readme.txt]: Can we add gutenberg? (are we allowed to mention other plugin names?)

File: README.md [hunk 11]
```diff
@@ -1 +1,9 @@
-# ClassicPress-Migration-Plugin
\ No newline at end of file
+# Switch to ClassicPress from WordPress
+
+Switch to ClassicPress from WordPress and say Goodbye to Gutenberg. (ClassicPress Migration Plugin)
```
Comment (scottybo) [inline README.md]: Please remove "and say Goodbye to Gutenberg"

File: README.md [hunk 12]
```diff
@@ -1 +1,9 @@
...
+
+https://www.ClassicPress.net/ is a new LTS fork actor of fomous CMS.
```
Comment (scottybo) [inline README.md]: Please follow the brand guidelines for the web address: https://www.classicpress.net/brand-guidelines/

Please change to "ClassicPress is a long term support (LTS) fork of WordPress 4.9.x with the goal of providing ongoing security patches, bug fixes and compatability updates for at least 2 years, giving you peace of mind that the WordPress you know and love can be used for the foreseeable future."

File: README.md [hunk 13]
```diff
@@ -1 +1,9 @@
...
+
+Available on GitHub @ https://github.com/ClassicPress/
+
+Sign this petition: https://www.change.org/p/petition-to-wordpress-no-gutenberg-in-wordpress-core/
```
Comment (scottybo) [inline README.md]: Please remove the petition text and link

File: .gitignore [hunk 14]
```diff
@@ -0,0 +1,19 @@
+*.log
+wp-config.php
```
Comment (scottybo) [inline .gitignore]: We don't need these Wordpress files and folders in this gitignore file

In fact, I don't think we need any of these?

File: switch-to-classicpress-from-wordpress.php (hunk 1 above)
Comment (marctenax) [reply switch-to-classicpress-from-wordpress.php]: OK removing.

File: switch-to-classicpress-from-wordpress.php (hunk 2 above)
Comment (marctenax) [reply switch-to-classicpress-from-wordpress.php]: OK start at v 1.0.0

File: switch-to-classicpress-from-wordpress.php (hunk 4 above)
Comment (marctenax) [reply switch-to-classicpress-from-wordpress.php]: Remove @version or all @version @pakage or remove @version 1.0.2 - 2018-08-30 and change it to @version 1.0.0 ?

File: switch-to-classicpress-from-wordpress.php (hunk 5 above)
Comment (marctenax) [reply switch-to-classicpress-from-wordpress.php]: Ok start at v 1.0.0

File: switch-to-classicpress-from-wordpress.php (hunk 6 above)
Comment (marctenax) [reply switch-to-classicpress-from-wordpress.php]: Why incorrect URL?

What is the correct URL?

File: switch-to-classicpress-from-wordpress.php (hunk 7 above)
Comment (marctenax) [reply switch-to-classicpress-from-wordpress.php]: Ok to remove "and say Goodbye to Gutenberg"

File: readme.txt (hunk 8 above)
Comment (marctenax) [reply readme.txt]: OK to remove versions after v 1.0.0

File: readme.txt (hunk 9 above)
Comment (marctenax) [reply readme.txt]: OK v 1.0.0 not need this.

File: readme.txt (hunk 10 above)
Comment (marctenax) [reply readme.txt]: It is considered Black Hat SEO and is not recommended initially.

Furthermore, only 5 tags are "really" allowed and published; the others are not visible, so it is better to insert only 5 of the most important ones initially.

https://developer.wordpress.org/plugins/wordpress-org/detailed-plugin-guidelines/#12-public-facing-pages-on-wordpress-org-readmes-must-not-spam/

File: README.md (hunk 11 above)
Comment (marctenax) [reply README.md]: Ok to remove "and say Goodbye to Gutenberg"

File: README.md (hunk 12 above)
Comment (marctenax) [reply README.md]: OK to change text

File: README.md (hunk 13 above)
Comment (marctenax) [reply README.md]: OK to remove the petition text and link

File: .gitignore (hunk 14 above)
Comment (marctenax) [reply .gitignore]: Totally remove .gitignore?

**Comment by marctenax** [comment]
Pull request closed.

**Comment by marctenax** [comment]
Please delete it, because is obsolete. Thanks!

**Pull request closed without merging**